packages/anasazi/test/OrthoManager/cxx_main_complex.cpp

```diff
@@ -127,8 +127,7 @@ int main(int argc, char *argv[])
     }
 
     // below we will assume that sizeX1 > sizeX2
-    // this does not effect our testing, since we will test P_{X1,Y1} P_{X2,Y2} as well as P_{X2,Y2} P_{X1,Y1}
```

Comment:
Why delete this line?

Reply:
This is copied over from the generalized test.  There isn't Y1, Y2 basis for this test, so the comment is not correct.